.changeset/strong-donkeys-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+fix: add 413 error page
```

Comment:
This would be enough :) Or if you'd like to keep to the conventional commit, it should be `feat`
```suggestion
Add 413 error page.
```

Reply:
@HeartSquared this has been updated.